example_github_workflow_files/automerge.yml

```diff
@@ -4,10 +4,17 @@ on:
   schedule:
     - cron: '05,17,29,41,53 * * * *'
   pull_request:
+    # Here we run when labels are applied, so that the "Override AutoMerge: name similarity is okay label is respected.
```

Comment:
```suggestion
    # labeled = run when labels are applied, so that the "Override AutoMerge: name similarity is okay label is respected.
    # synchronize = run when a commit is pushed to the PR
```

Reply:
@ericphanson Does `synchronize` also cover when the PR is first opened, or do we also need to add `opened` to the list?

Reply:
Hm, I think we may need to add `opened` as well